iocore/net/SSLUtils.cc

```diff
@@ -418,6 +418,61 @@ PerformAction(Continuation *cont, const char *servername)
   return SSL_TLSEXT_ERR_OK;
 }
 
+#if TS_USE_HELLO_CB
+// Pausable callback
+static int
+ssl_client_hello_callback(SSL *s, int *al, void *arg)
+{
+  const char *servername = nullptr;
+  const unsigned char *p;
+  size_t remaining, len;
+  if (SSL_client_hello_get0_ext(s, TLSEXT_TYPE_server_name, &p, &remaining) || remaining <= 2) {
```

Comment:
Is `SSL_client_hello_get0_ext` only present if the Client HELLO callback is also present? Would this compile without the `#if`?

Reply:
Only valid with openssl 1.1.1 as well so needs to be in the client hello callback ifdef.